### PR TITLE
Add option for SSL

### PR DESCRIPTION
**Pull request by alisa-aylward-toast**
# Description of change
This will allow people to pass in an optional SSL parameter for when they do a Redshift set up. 

# Manual QA steps
Initially:
Screen Shot 2019-11-18 at 7 53 11 PM
After:
Screen Shot 2019-11-19 at 9 39 53 AM
(didn't want to put any of our data, but we it did run)
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Comment by cmerrick** [comment]
Hi @alisa-aylward-toast, thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/). It's all electronic and will take just minutes.

File: tap_redshift/__init__.py [hunk 1]
```diff
@@ -18,22 +18,20 @@
...
 
 import copy
+import datetime
+import ssl
+import sys
 import time
 from itertools import groupby
 
 import pendulum
-import datetime
```
Comment (alisa-aylward-toast) [inline tap_redshift/__init__.py:6]: The reason these moved is because I used isort to sort them in accordance with pep8.

**Comment by alisa-aylward-toast** [comment]
@cmerrick   Thanks! my status says "Sent"

**Comment by cmerrick** [comment]
You did it @alisa-aylward-toast!

Thank you for signing the Singer Contribution License Agreement.

**Review by dmosorast — APPROVED** [review]
Thanks, @alisa-aylward-toast! The code looks good. Just a few minor descriptive changes that I'll add post-merge.

**Pull request merged**